gtest_parallel.py

```diff
@@ -270,7 +270,7 @@ def __register_exit(self, task):
     self.times.record_test_time(task.test_binary, task.test_name,
                                 task.last_execution_time)
     if self.test_results:
-      self.test_results.log(task.test_name, task.runtime_ms,
+      self.test_results.log(task.test_name, int(task.runtime_ms / 1000),
```

Comment:
This truncation seems undesirable (for very fast tests 0/1 seconds is not very helpful), can this print 0.543 seconds for instance?

Reply:
Done, I think decimal should work fine.